### PR TITLE
Improve the readme

### LINKED ISSUE #7: readme: add a note about wait-for-services.

### PR DESCRIPTION
**Pull request by yakutovicha**
fixes #7 
* Fix the description of git branch vs docker tag correspondance.
* Add information about wait-for-services.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,18 +4,27 @@ This repository adds
...
+# Docker image
 
 The docker image contains:
  * [PostgreSQL](https://www.postgresql.org/)
  * [RabbitMQ](https://www.rabbitmq.com/)
- * Configured Linux environment for $SYSTEM_USER user
+ * Configured Linux environment for $SYSTEM_USER.
+
+## Wait for services
+This image provides a mechanism to wait untill all the services were launched by the statup scripts. However, it does not
```
Comment (ltalirz) [inline README.md]:
```suggestion
This image provides a mechanism to wait until the startup script has *launched* all services.
```

File: README.md [hunk 2]
```diff
@@ -4,18 +4,27 @@ This repository adds
...
+check that services did start. Example of usage:
```
Comment (ltalirz) [inline README.md]:
```suggestion
Example of usage:
```

File: README.md [hunk 3]
```diff
@@ -4,18 +4,27 @@ This repository adds
...
+
+```
+$ docker exec --tty $DOCKERID wait-for-services
+```
+This command will exit when all startup scripts are done.
```
Comment (ltalirz) [inline README.md:22]:
```suggestion
This command will exit when all startup scripts are done.
Note, however, that the mechanism does not check that all services are actually running.
```

**Review by ltalirz — CHANGES_REQUESTED** [review]
thanks @yakutovicha, some minor suggestions

File: README.md [hunk 4]
```diff
@@ -1,21 +1,57 @@
...
+Note, however, that the mechanism does not check that all services are actually running.
+
+## Docker ARGs
+The the following arguments can be set during *build* time:
+```
+ARG NB_USER="aiida"
+ARG NB_UID="1000"
+ARG NB_GID="1000"
+```
```
Comment (ltalirz) [inline README.md:31]: Perhaps add:
These define the *default* user name, user id and group id for containers launched from the image.

Comment (yakutovicha) [reply README.md:31]: done

**Review by ltalirz — APPROVED** [review]
thanks @yakutovicha

**Pull request merged**